app/models/manageiq/providers/kubernetes/container_manager_mixin.rb

```diff
@@ -41,9 +41,9 @@ def kubernetes_version
 
   PERF_ROLLUP_CHILDREN = :container_nodes
 
-  def hawkular_status
+  def hawkular_availability
     client = ManageIQ::Providers::Kubernetes::ContainerManager::MetricsCapture::HawkularClient.new(self)
-    client.status
+    client.metrics.avail.get_data('all', :limit => 1)
```

Comment:
@yaacov please add a clear comment here explaining what we're doing and why we're doing it (e.g. Here we want to make sure that the token... bla bla...)